pkg/cmd/cli/cmd/login.go

```diff
@@ -178,19 +181,68 @@ func (o LoginOptions) Validate(args []string, serverFlag string) error {
 	return nil
 }
 
-// RunLogin contains all the necessary functionality for the OpenShift cli login command
-func RunLogin(cmd *cobra.Command, options *LoginOptions) error {
-	if err := options.GatherInfo(); err != nil {
+func (o *LoginOptions) RunLogin() error {
+	if !o.Force && o.isLikelyAdminKubeConfig() {
+		return errors.New("It looks like your config file is used for a cluster-admin (system:admin).  You probably shouldn't modify this file using `oc login`.\n" +
+			"Modifying the cluster-admin configuration file can result in confusion when you try to use it later for privileged actions and get denied.\n" +
```

Comment:
`"Modifying the cluster-admin configuration file can result in removal of privileges from the cluster-admin role."`

?